PEP8 Standards Fixes

Dmchale92 opened the pull request:
--------- BEGIN META VARIABLES --------
Project : dropbox-sdk-python
Task : PEP8 Standards Fixes
Done : 3-16-17
--------- END META VARIABLES --------

Dmchale92 committed:
PEP8 Standards Fixes

--------- BEGIN META VARIABLES --------
Project : dropbox-sdk-python
Task : PEP8 Standards Fixes
Done : 3-16-17
--------- END META VARIABLES --------

posita commented on example/back-up-and-restore/backup-and-restore-example.py:
```diff
@@ -37,44 +48,62 @@ def backup():
     print("Finding available revisions on Dropbox...")
-    entries = dbx.files_list_revisions(BACKUPPATH, limit=30).entries  # pylint: disable=no-member
+    entries = dbx.files_list_revisions(
+        BACKUPPATH, limit=30).entries  # pylint: disable=no-member
```
Breaking up this line confuses the linter. Consider instead:

```py
revisions = dbx.files_list_revisions(BACKUPPATH, limit=30)
entries = revisions.entries  # pylint: disable=no-member
```

posita reviewed (changes requested):
Thanks for the contribution!

Please consider updating the [`flake8` section of `tox.ini`](https://github.com/dropbox/dropbox-sdk-python/blob/master/tox.ini#L17-L20) and the [`MESSAGES CONTROL` section of `.pylintrc`](https://github.com/dropbox/dropbox-sdk-python/blob/master/.pylintrc#L1-L3) to reflect the changes. This will likely mean turning back on certain linter options, but some experimentation may be necessary.

Dmchale92 committed:
Breaking up entry revision retrieval to address linter confusion

--------- BEGIN META VARIABLES --------
Project : dropbox-sdk-python
Task : Cleaning up backup-and-restore-example.py
Done : 3-16-17
--------- END META VARIABLES --------

Dmchale92 committed:
Fixing E261 - at least two spaces before inline comment

--------- BEGIN META VARIABLES --------
Project : dropbox-sdk-python
Task : Addressing linter error on travis CI build for prior commit to
backup-and-restore-example.py
Done : 3-16-17
--------- END META VARIABLES --------

Dmchale92 committed:
--------- BEGIN META VARIABLES --------
Project : dropbox-sdk-python
Task : Updating tox.ini to ignore E1101 (no-member)
Done : 3-16-17
--------- END META VARIABLES --------

Dmchale92 commented:
Broke up the revision retrieval code, and fixed a subsequent E261 error. Also added E1101 to tox.ini, it looks like MESSAGES CONTROL for .pylintrc already has locally-disabled disabled.

posita commented:
@Dmchale92, it looks good. If you can squash commits, I'll merge. Thanks!

(As an aside, [`ez_setup.py`](https://bootstrap.pypa.io/ez_setup.py) comes from elsewhere (formerly from [PEAK](http://peak.telecommunity.com/), now from [PyPA](https://www.pypa.io/en/latest/). We might consider removing it altogether in the future and leave bootstrapping `setuptools` up to the installer. I think most modern versions of Python already include it anyway.)

posita reviewed (changes requested):
@Dmchale92, it looks good. If you can squash commits, I'll merge. Thanks!

posita commented:
As an aside, [`ez_setup.py`](https://bootstrap.pypa.io/ez_setup.py) comes from elsewhere (formerly from [PEAK](http://peak.telecommunity.com/), now from [PyPA](https://www.pypa.io/en/latest/)). We might consider removing it altogether in the future and leave bootstrapping `setuptools` up to the installer. I think most modern versions of Python already include it anyway.

rogebrd commented:
Hello, If you are still interested in updating this piece of code, please acknowledge this message and make the necessary updates.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dropbox/dropbox-sdk-python?pullRequest=97) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/dropbox/dropbox-sdk-python?pullRequest=97) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dropbox/dropbox-sdk-python?pullRequest=97) it.</sub>

AndreyVMarkelov commented:
Thanks for the contribution. This patch predates the current generated SDK, supported Python versions, and repository tooling, and includes changes to files that no longer exist. It is no longer applicable to the current codebase, so closing as obsolete.

dmchaledev commented:
I was just wondering whatever happened to this one. Glad it's resolved!
